Add files via upload

DavidW7 opened the pull request.

DavidW7 committed:
Add files via upload

grig-guz reviewed (commented):
First, for which dataset is this?

grig-guz reviewed (changes requested):
Can you please move this to data folder?

DavidW7 committed:
Update audiotxtmatching.py

DavidW7 committed:
Update and rename audiotxtmatching.py to librispeechparsing.py

Script for parsing the LibriSpeech dataset folder to match the audio file names with its corresponding texts
